consensus/polybft/transport.go

```diff
@@ -0,0 +1,117 @@
+package polybft
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/0xPolygon/go-ibft/messages/proto"
+	pbftproto "github.com/0xPolygon/polygon-edge/consensus/polybft/proto"
+	"github.com/0xPolygon/polygon-edge/network"
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/hashicorp/go-hclog"
+	"github.com/libp2p/go-libp2p/core/peer"
+)
+
+// Transport is an abstraction of network layer for a bridge
+type BridgeTransport interface {
+	Multicast(msg interface{})
+}
+
+type runtimeTransportWrapper struct {
+	bridgeTopic *network.Topic
+	logger      hclog.Logger
+}
+
+var _ BridgeTransport = (*runtimeTransportWrapper)(nil)
+
+// Multicast publishes any message as pbftproto.TransportMessage
+func (g *runtimeTransportWrapper) Multicast(msg interface{}) {
+	data, err := json.Marshal(msg)
+	if err != nil {
+		g.logger.Warn("failed to marshal bridge message", "err", err)
+
+		return
+	}
+
+	err = g.bridgeTopic.Publish(&pbftproto.TransportMessage{Data: data})
+	if err != nil {
+		g.logger.Warn("failed to gossip bridge message", "err", err)
+	}
+}
+
+// subscribeToBridgeTopic subscribes for bridge topic
+func (cr *consensusRuntime) subscribeToBridgeTopic(topic *network.Topic) error {
```

Comment:
I know these methods are transport related, but it maybe it makes more sense to put methods to corresponding files (e.g. `consensusRuntime` related to `consensus_runtime.go`, `polybft` related to `polybft.go` etc.)...?

Reply:
I thought about that, but there is so much code in those files already. Do you agree to finish the whole feature, see how the code will be organised and than we can transfer this code if we see that is easier to follow data-flow?

Reply:
👍